pkg/services/live/channel.go

```diff
@@ -1,27 +1,40 @@
 package live
 
 import (
-	"fmt"
 	"strings"
 )
 
-// ChannelIdentifier is the channel id split by parts
-type ChannelIdentifier struct {
-	Scope     string // grafana, ds, or plugin
-	Namespace string // feature, id, or name
-	Path      string // path within the channel handler
+// ChannelAddress is the channel id split by parts
+type ChannelAddress struct {
+	Scope     string `json:"scope,omitempty"`     // grafana, ds, or plugin
+	Namespace string `json:"namespace,omitempty"` // feature, id, or name
+	Path      string `json:"path,omitempty"`      // path within the channel handler
```

Comment:
The comments should go above and be on sentence form. When we enable better linting, it will catch this.